pkg/apis/crds/karpenter.sh_nodepools.yaml

```
@@ -59,11 +59,11 @@ spec:
                             type: string
                           nodes:
                             default: 10%
-                            description: 'Nodes dictates how many NodeClaims owned by this NodePool can be terminating at once. It must be set. This only considers NodeClaims with the karpenter.sh/disruption taint. We can''t use an intstr.IntOrString since kubebuilder doesn''t support pattern checking for int nodes for IntOrString nodes. Ref: https://github.com/kubernetes-sigs/controller-tools/blob/55efe4be40394a288216dab63156b0a64fb82929/pkg/crd/markers/validation.go#L379-L388'
+                            description: 'Nodes dictates the maximum number of NodeClaims owned by this Node that can be terminating at once. This is calculated by counting nodes that have a deletion timestamp set, or are marked for deletion in memory. This field is required. This cannot be of type intstr.IntOrString since kubebuilder doesn''t support pattern checking for int nodes for IntOrString nodes. Ref: https://github.com/kubernetes-sigs/controller-tools/blob/55efe4be40394a288216dab63156b0a64fb82929/pkg/crd/markers/validation.go#L379-L388'
                             pattern: ^((100|[0-9]{1,2})%|[0-9]+)$
                             type: string
                           schedule:
-                            description: Schedule specifies when a budget begins being active, using the upstream cronjob syntax. If omitted, the budget is always active. Currently timezones are not supported. This is required if Duration is set.
+                            description: Schedule specifies when a budget begins being active, following the upstream cronjob syntax. If omitted, the budget is always active. Like in upstream, timezones are not supported. This is required if Duration is set.
```

```suggestion
                            description: Schedule specifies when a budget begins being active. If omitted, the budget is always active. Timezones are not supported. This field is required if Duration is set.
```

You say upstream a few times and I feel like we don't need to say that we are following someone else. I get that this might make it easier to reason about, but we effectively _are_ upstream now, so we are just following convention in general